ts/components/bottomSheet/BottomSheetHeader.tsx

```diff
@@ -37,34 +40,43 @@ type Props = {
 export const BottomSheetHeader: React.FunctionComponent<Props> = ({
   title,
   onClose
-}: Props) => (
-  <View style={styles.row}>
-    {React.isValidElement(title) ? (
-      title
-    ) : (
-      <View
-        style={IOStyles.flex}
+}: Props) => {
+  const headerRef = React.createRef<View>();
+
+  useEffect(() => {
+    setAccessibilityFocus(headerRef, 1000 as Millisecond);
+  }, [headerRef]);
+
+  return (
+    <View style={styles.row}>
+      {React.isValidElement(title) ? (
+        <View ref={headerRef}>{title}</View>
+      ) : (
+        <View
+          ref={headerRef}
+          style={IOStyles.flex}
+          accessible={true}
+          accessibilityRole={"header"}
+          accessibilityLabel={typeof title === "string" ? title : undefined}
+        >
+          <H3>{title}</H3>
+        </View>
+      )}
```

Comment:
In `CGN_MERCHANTS_DETAIL`, if we try to open a bottomsheet we receive:

<img width="200" alt="Schermata 2022-04-21 alle 16 47 34" src="https://user-images.githubusercontent.com/26501317/164484466-cd53683a-e10d-4601-9b22-1a1f7286c2b0.png">

I think this could be caused by the conditional ref assignment. I've tried to move the ref to the parent component and seems fine. Here my try:

```suggestion
    <View style={styles.row} ref={headerRef}>
      {React.isValidElement(title) ? (
        <View>{title}</View>
      ) : (
        <View
          style={IOStyles.flex}
          accessible={true}
          accessibilityRole={"header"}
          accessibilityLabel={typeof title === "string" ? title : undefined}
        >
          <H3>{title}</H3>
        </View>
      )}
```